source/GarminSDDataHandler.mc

```diff
@@ -177,8 +176,8 @@ class GarminSDDataHandler {
       //System.println("DataHandler - sending Accel Data");
       //writeLog("DataHandler.accelCallback()","Sending accel Data");
       mComms.sendAccelData();
-      //Ui.requestUpdate();
     }
+    Ui.requestUpdate();
```

Comment:
I am starting to remember what this is doing.   In this position (or its original one slightly higher in the function), it updates the user interface and re-draws the screen every second.   The commented out version of this call just after the mComms.sendAccelData() would update the display every 5 seconds instead.   I thought that re-drawing the screen might have been draining the battery - but I don't think it made a noticeable difference, which is why it is left re-drawing every second.